apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -195,12 +195,7 @@ class ShareDialog extends React.Component {
 
   render() {
     let image;
-    let modalClass = 'modal-content';
-    if (this.props.icon) {
-      image = <img className="modal-image" src={this.props.icon}/>;
-    } else {
-      modalClass += ' no-modal-icon';
-    }
+    let modalClass = 'modal-content no-modal-icon';
```

Comment:
Apologies for the late feedback, but I don't like that we're keeping the `this.props.icon` around, but then not showing the icon when it is set. One reason this is bad is that it breaks all of the storybooks for ShareDialog which still show the icon.

Could you please instead keep the code here the same as it appears in staging, and simply remove the `icon={appOptions.skin.staticAvatar}` line from header.js instead?